settings.gradle

```
@@ -7,7 +7,8 @@ pluginManagement {
 }
 
 rootProject.name = "opentelemetry-java-examples"
-include ":opentelemetry-examples-autoconfigure",
+include ":opentelemetry-examples-agent",
```

to help avoid confusion with opentelemetry collector "agent"
```suggestion
include ":opentelemetry-examples-javaagent",
```